docs/content/direct/deploy-on-k3d.md

```diff
@@ -81,13 +81,13 @@ In addition to [pre-reqs](pre-reqs.md), install k3d v5.6.0 (only k3d version tes
 
     Run transport deployment script (in `scripts/deploy-transport-controller.sh`), as follows.
     This script requires that the user's current kubeconfig context be for the kubeflex hosting cluster.
-    This script expects to get two or three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
+    This script expects to get two or three arguments - (1) wds name; (2) its name; and (3) transport controller image.  
```

Comment:
Do we want to turn space names to full-capitalized to not confuse its (inventory transport space) with the word `its`? In documentation at least.

Other than this, I reviewed all renamings and cloned your branch to verify that imbs no longer exists - all good.

Reply:
Yes, initialisms should be capitalized.

Reply:
Done